website/static/js/registrationRetraction.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * On page load, focuses on justification input and
+ * maintains knockout ViewModel
+**/
+'use strict';
+
+var ko = require('knockout');
+require('knockout.validation');
+var $ = require('jquery');
+
+var $osf = require('js/osfHelpers');
+
+var RegistrationRetractionViewModel = function(submitUrl, registrationTitle) {
+
+    var self = this;
+
+    // Custom Validation
+    ko.validation.rules['mustEqual'] = {
+        validator: function (val, otherVal) {
+            return val === otherVal;
+        },
+        message: "The field does not match the required input."
```

Comment:
As per our `.jshintrc`, use singlequotes for strings (makes searching for strings easier).